### ISSUE TITLE
xlnet large 

### ISSUE THREAD

**Issue by gogokre** [huggingface/transformers#3977]
If you train in batch 8 at max length = 512 with the xlnet-base-cased, you will learn well with changes in values such as Loss, Accuracy, Precision, Recall, and F1-score.

However, if you train with batch 2 at max length = 512 with a xlnet-large-cased, even if you run several times, there is almost no change, and precision is always 1.00000, and other values are only fixed.

Is it because the batch size is too small? Help.

[epoch1]
Tr Loss | Vld Acc | Vld Loss | Vld Prec | Vld Reca | Vld F1
0.39480 | 0.87500 | 0.39765 | 1.00000 | 0.87500 | 0.93333

[epoch2]
  Tr Loss | Vld Acc | Vld Loss | Vld Prec | Vld Reca | Vld F1
0.39772 | 0.87500 | 0.38215 | 1.00000 | 0.87500 | 0.93333

**Comment by FacingBugs** [comment]
Maybe the task is learning rate and batch size sensitive. You can try to decrease the learning rate when changing the batch size from 8 to 2.

**Comment by gogokre** [comment]
> Maybe the task is learning rate and batch size sensitive. You can try to decrease the learning rate when changing the batch size from 8 to 2.

Using xlnet-large-cased with batch size 2 and learning rate = 1e-5 is the same symptom. Is there another way?
